navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/snap/SnapToRoute.java

```diff
@@ -62,7 +75,8 @@ private static float snapLocationBearing(RouteProgress routeProgress) {
       TurfConstants.UNIT_METERS);
     // Measure 1 meter ahead of the users current location
     Point futurePoint = TurfMeasurement.along(
-      lineString, routeProgress.currentLegProgress().currentStepProgress().distanceTraveled() + 1,
+      lineString,
+      routeProgress.currentLegProgress().currentStepProgress().distanceTraveled() + 1,
```

Comment:
When we reach the end of the current step, will Turf continue to measure a meter ahead continuing to the next step?  I'm wondering if this is where we are seeing the bearing issues when switching steps